app/controllers/DistributionController.scala

```diff
@@ -65,15 +66,15 @@ class DistributionController @Inject() (
             for {
               updatedAnswers <-
                 Future.fromTry(
-                  if (value == Distribution.No) {
-                    request.userAnswers
-                      .set(DistributionPage, value)
-                      .flatMap(answer =>
+                  request.userAnswers
+                    .set(DistributionPage, value)
+                    .flatMap(answer =>
+                      if (value == Distribution.No) {
                         answer.set(DistributionsIncludedPage, DistributionsIncludedForm(DistributionsIncluded.No, None))
```

Comment:
Why not 

```
answer.remove(DistributionsIncludedPage) 
```

We record it as No, but they never said no to the second question

Reply:
That doesn't work when you get through to the check your answer page, if you remove the DistributionsIncludedPage values then the section isn't shown causing the error in https://jira.tools.tax.service.gov.uk/browse/MRC-39